workflows/linters.yml

```diff
@@ -0,0 +1,87 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+<<<<<<< HEAD:workflows/linters.yml
+  lighthouse:
```

Comment:
- [ ] [REQUIRED] Please, make sure that you use the correct linter file. Since this file is corrupt as it shows here on line `9`.